doc/en/components/charts/features/chart-axis-gridlines.md

```diff
@@ -22,45 +26,66 @@ Axis major gridlines are long lines that extend horizontally (y-axis) or vertica
 
 <div class="divider--half"></div>
 
-## Axis Gridlines Properties
+## $PlatformShort$ Axis Gridlines
+
+Setting the axis major interval property specifies how often major gridlines and axis labels are rendered on an axis. Similarly, the axis minor interval property specifies how frequent minor gridlines are rendered on an axis. These properties must be set to a value greater than 0.
+
+In order to display minor gridlines that correspond to minor interval, you need to set `MinorStroke` and `MinorStrokeThickness` properties on the axis. This is because minor gridlines do not have a default color or thickness and they will not be displayed without first assigning them.
+
+You can customize how the gridlines are displayed in your $PlatformShort$ chart by setting the following properties:
+
+- `MajorStrokeColor`: This property sets the color of axis major gridlines.
+- `MinorStrokeColor`: This property sets the color of axis minor gridlines.
+- `MajorStrokeThickness`: This property sets the thickness in pixels of the axis major gridlines.
+- `MinorStrokeThickness`: This property sets the thickness in pixels of the axis minor gridlines.
+- `MajorInterval`: This value provides adequate spacing for axis labels and major gridlines, if used. Note that the interval for axis labels will also be set by this value, displaying one label at the point on the axis associated with the interval. 
```

Comment:
these API members do not exist on CategoryChart and you cannot use API members of DataChart in topic that talks about CategoryChart. Please add back the table with correct API members and append a new column with description of those properties.